app/controllers/api/v1/search_controller.rb

```diff
@@ -688,9 +688,9 @@ def self.match_results_by_filter(search_result:, result_key:, facets:)
         if db_facet.is_numeric?
           match = matching_facet[:filters].dup
           match.delete(:name)
-          return match
+          match
```

Comment:
Just to make sure I understand it - having an explicit `return` is not preferred by Ruby right?

Reply:
Yes - all Ruby methods implicitly return the last object operated on, so just calling `foo` will return whatever value is assigned to `foo`.  Calling `return` is only required when you need to explicitly halt execution and return a value.